content/misc/lab-setup.md

```diff
@@ -0,0 +1,24 @@
+# Setting up the Lab environment
+
+If you have already cloned the repository, make sure it is updated:
+
+```console
+cd operating-systems
+git pull --rebase
+```
+
+The command may fail if you have uncommitted changes.
+If that is the case, commit changes and retry.
+
+If you haven't already cloned the repository, do it and enter the repository:
+
+```console
+git clone https://github.com/open-education-hub/operating-systems
+cd operating-systems
+```
+
+Navigate to the chapter lab directory:
+
+```console
+cd content/chapters/software-stack/lab/
+```
```

Comment:
The Docker container defines some aliases to navigate more quickly [1]. I still think they're pointless, but maybe we should mention them here. @Alex-deVis would love this.

[1] https://github.com/open-education-hub/operating-systems/blob/b6a0cbdff828bdabfb7403a597e325c747e6bfbe/util/os-runner/config_terminal.sh#L21-L25